cmd/govim/gopls_client.go

```diff
@@ -145,6 +146,9 @@ func (g *govimplugin) Configuration(ctxt context.Context, params *protocol.Param
 	if g.vimstate.config.CompletionBudget != nil {
 		conf[goplsCompletionBudget] = *config.CompletionBudget
 	}
+	if g.vimstate.config.ExperimentalTempModfile != nil {
+		conf[goplsTempModfile] = *g.vimstate.config.ExperimentalTempModfile
```

Comment:
Should it be `*config.ExperimentalTempModfile` for consistency?

Reply:
It should, thank you :)